src/spartan/ppsnark.rs

```diff
@@ -810,8 +868,7 @@ pub struct RelaxedR1CSSNARK<E: Engine, EE: EvaluationEngineTrait<E>> {
   eval_ts_col: E::Scalar,
 
   // a PCS evaluation argument
-  eval_arg_W: EE::EvaluationArgument,
-  eval_arg_batch: EE::EvaluationArgument,
+  eval_arg: EE::EvaluationArgument,
```

Comment:
This might be assuming the the inclusion of sum-check related to padding checks of W.